### PR TITLE
✨ feat: add auditing Timestamp class

### PR DESCRIPTION
**Pull request by ruh0n**
### What is the PR type?
- [x] : feature
- [ ] : refactor
- [ ] : bug
- [ ] : documentation
- [ ] : chore
- [ ] : displace
- [ ] : etc

### What does this PR do?

- Add Timestamp.java
- Enable JPA auditing

### CODE REVIEW

**Comment by devJWL** [comment]
LGTM

**Review by devJWL — APPROVED** [review]
LGTM

File: src/main/java/com/ptjcoding/nbcampspringnewsfeed/NbcampSpringNewsfeedApplication.java [hunk 1]
```diff
@@ -4,7 +4,10 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+@EnableJpaAuditing()
```
Comment (Modafi) [inline src/main/java/com/ptjcoding/nbcampspringnewsfeed/NbcampSpringNewsfeedApplication.java]: () 괄호에 파라미터가 없으면 생략할 수 있지 않을까요?

- https://docs.spring.io/spring-data/jpa/reference/auditing.html
``` java
@Configuration
@EnableJpaAuditing
class Config {

  @Bean
  public AuditorAware<AuditableUser> auditorProvider() {
    return new AuditorAwareImpl();
  }
}
```

**Review by gloz0315 — APPROVED** [review]
LGTM

**Pull request merged**